sass/directives/_sections.scss

```diff
@@ -0,0 +1,20 @@
+@mixin logos-section {
```

Comment:
I think I tend to agree, here. For us to include something in `style-base`, we either have to have thought through re-usability ahead of time to some degree, or we have to let it bake inside `employer` first - until we have enough use cases that have morphed it into a somewhat generic component.